### PR TITLE
UI eus 223 flag uploaded reports enter data

### PR DESCRIPTION
**Pull request by elsenhans**
https://issues.folio.org/browse/UIEUS-223

### CODE REVIEW

File: src/components/ReportUpload/CounterUploadModal/CounterUpload.js [hunk 1]
```diff
@@ -132,17 +148,24 @@ class CounterUpload extends React.Component {
     });
   };
 
+  cancleUpload = () => {
+    this.closeInfoModal();
+    this.setState({
+      selectedFile: {},
```
Comment (rchr) [inline src/components/ReportUpload/CounterUploadModal/CounterUpload.js:154]: I think it also needed to clear the state of `reportEditedManually` and `editReason`.

I tried to cancel the upload with given `reportEditedManually` and `editReason`. After trying to upload a report again, the `editReason` specified before cancel was used but not shown.

File: test/bigtest/tests/details-udp-test.js [hunk 2]
```diff
@@ -266,9 +266,67 @@ describe('UDPDetailsPage', () => {
...
+      });
+      it('does render edit manually fields', () => {
+        expect(udpDetailsPage.uploadCounterModal.reportEditedManuallyCheckbox.isPresent).to.equal(true);
+        expect(udpDetailsPage.uploadCounterModal.editReasonTextfield.isPresent).to.equal(true);
+      });
+      it('does render cancel button', () => {
+        expect(udpDetailsPage.uploadCounterModal.cancelUploadCounterReportButton.isPresent).to.equal(true);
+      });
+
+      describe('handling drop counter report', () => {
```
Comment (rchr) [inline test/bigtest/tests/details-udp-test.js:279]: Nice work! :)

**Review by rchr — CHANGES_REQUESTED** [review]
Nice work! Looks good, except one small thing.

In general I prefer functional components over class components, see `ReportUpload.js`. However, we can keep it as it is now.

File: src/components/ReportUpload/CounterUploadModal/CounterUpload.js (hunk 1 above)
Comment (elsenhans) [reply src/components/ReportUpload/CounterUploadModal/CounterUpload.js:155]: Thanks for the feedback! Did not realized that behaviour.

File: test/bigtest/tests/details-udp-test.js (hunk 2 above)
Comment (elsenhans) [reply test/bigtest/tests/details-udp-test.js:279]: Thanks :)

**Review by rchr — APPROVED** [review]

**Pull request merged**